src/components/Capture/Face.tsx

```diff
@@ -108,19 +125,26 @@ class Face extends Component {
       pageId,
     } = this.props
     const title = translate('selfie_capture.title')
-    const props = {
-      onError: this.handleError,
+
+    const cameraProps: Omit<CameraProps, 'buttonType'> = {
       ...this.props,
-    }
-    const cameraProps = {
       renderTitle: <PageTitle title={title} smaller />,
-      cameraClassName: style.faceContainer,
       renderFallback: isDesktop
         ? this.renderCrossDeviceFallback
         : this.renderUploadFallback,
-      inactiveError: getInactiveError(this.isUploadFallbackDisabled()),
       isUploadFallbackDisabled: this.isUploadFallbackDisabled(),
-      ...props,
+    }
+
+    const withLocalisedProps: WithLocalisedProps = {
+      ...this.props,
+    }
+
+    const withTrackingProps: WithTrackingProps = {
+      ...this.props,
+    }
+
+    const stepComponentFaceProps: StepComponentFaceProps = {
+      ...this.props,
```

Comment:
What's the purpose of those three? I think we can just use `{...this.props}` in the components below

Reply:
I was overwhelmed with props drilling during the migration. I'm putting back `...this.props` 😅